code/game/atoms.dm

```diff
@@ -247,7 +247,8 @@
 
 // Don't make these call bicon or anything, these are what bicon uses. They need to return an icon.
 /atom/proc/examine_icon()
```

Comment:
Doing this could interfere with procs like on the gargoyle. it called . = ..()

In my tests, we could drop almost all examine_icon. Only relevant might be the gargoyle.